app/src/main/java/com/kickstarter/libs/AnalyticEvents.kt

```diff
@@ -956,6 +929,13 @@ class AnalyticEvents(trackingClients: List<TrackingClientType?>) {
         client.track(CAMPAIGN_DETAILS_BUTTON_CLICKED, experimentProperties(projectData))
     }
 
+    fun trackCampaignDetailsCTAClicked(projectData: ProjectData) {
+        val props: HashMap<String, Any> = hashMapOf(CONTEXT_PAGE.contextName to CAMPAIGN_DETAILS.contextName)
```

Comment:
I think this is the wrong value, since we're tapping the CTA on the project page. I think the key should be `project`

Reply:
Yeah noticed that too. Thanks for pointing that out. Corrected